masonry/src/properties/background.rs

```diff
@@ -19,12 +19,17 @@ pub enum Background {
     Gradient(Gradient),
 }
 
-impl Property for Background {}
+impl Property for Background {
+    fn static_default() -> &'static Self {
+        // This matches the CSS default.
+        const DEFAULT: Background = Background::Color(AlphaColor::TRANSPARENT);
```

Comment:
Elsewhere this is a `static` and here a `const` ... I think this should be `static` ?

Reply:
but also, this is a super cheap value, so I don't know what the gain here is.

Reply:
> Elsewhere this is a static and here a const ... I think this should be static ?

`const` and `static` are essentially equivalent here, since the compiler will store `const` values in statics anyway. But yeah, it worth keeping the code consistent.

> but also, this is a super cheap value, so I don't know what the gain here is.

Basically, the line of reasoning is:

- We want `Properties[/Ref/Mut]::get()` to return a `T`, not an `Option<T>`.
- There is no `Default` impl for `&T`. So we need a `Property::static_default()` method that will return a `&'static T` we can use anywhere.
- The way we do that is by creating a static inside each impl, and returning a reference to that static.

I guess I could document that.